assets/js/actions/index.js

```diff
@@ -41,7 +41,7 @@ function onNavigateTo (model) {
       redirectToLogin()
       return
     }
-    if (model.routeName === 'Item' && params.id) {
+    if (model.routeName === 'Account' && params.id) {
```

Comment:
rename `params.id` to `params.address` for more clarity.
This will require changes in the `settings` object as well (`routeDesc`)